eng/.docsettings.yml

```diff
@@ -165,7 +166,7 @@ known_content_issues:
   - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
   - ['sdk/servicebus/Microsoft.Azure.ServiceBus/README.md','#5499']
   - ['sdk/eventhub/Microsoft.Azure.EventHubs/README.md','#5499']
-  - ['sdk/eventhub/Azure.Messaging.EventHubs/README.md','#5499']
+  - ['sdk/alertsmanagement/Microsoft.Azure.Management.AlertsManagement/README.md', '#5499']
```

Comment:
Not sure when this got added, but a rebase to master picked it up and it was causing analyze failures.